welcome/index.adoc

```diff
@@ -55,6 +55,18 @@ ifdef::openshift-online,openshift-aro[]
 Start with **xref:../architecture/architecture.adoc#architecture-overview-architecture[Architecture]**.
 endif::[]
 
+ifdef::microshift[]
+Start with xref:../microshift_getting_started/microshift-understanding.adoc#microshift-understanding[Understanding {product-title}] and
+xref:../microshift_install/microshift-install-rhel-for-edge.adoc#microshift-install-rhel-for-edge[Installing].
+Then, see the
+xref:../microshift_release_notes/microshift-4-12-release-notes.adoc#microshift-4-12-release-notes[release notes].
+
+[IMPORTANT]
+====
+Red Hat might not provide or support an update or upgrade path from Developer Preview or Technology Preview versions to the Generally Available version of {product-title}. A new installation might be necessary.
```

Comment:
```suggestion
Red Hat will not provide or support an update or upgrade path from Developer Preview or Technology Preview versions to later versions of {product-title}. A new installation will be necessary.
```

@DanielFroehlich I'm proposing some stronger language here, changing "might not" to "will not". Can you sign off on that?

@ShaunaDiaz I changed "the Generally Available version" to "later versions" because we wouldn't support upgrading from dev preview to tech preview either. I tried to find wording to be more explicit, but that made the sentence unparseable. If you have other wording that conveys the same meaning, that's fine with me.

Reply:
/lgtm